vars/piperPipelineStageAcceptance.groovy

```diff
@@ -6,6 +6,7 @@ import groovy.transform.Field
 import static com.sap.piper.Prerequisites.checkScript
 
 @Field String STEP_NAME = getClass().getName()
+@Field String STAGE_NAME = 'endToEndTests'
```

Comment:
I think if this is the technical stage name we should name it like that.
```suggestion
@Field String TECHNICAL_STAGE_NAME = 'endToEndTests'
```

Reply:
OK